library/src/cache/patch_manager.rs

```diff
@@ -261,6 +273,15 @@ impl PatchManager {
             .unwrap_or(false)
     }
 
+    /// Whether this patch is in the process of booting.
+    fn is_currently_booting_patch(&self, patch_number: usize) -> bool {
+        if let Some(currently_booting_patch) = &self.patches_state.currently_booting_patch {
+            return currently_booting_patch.number == patch_number;
+        } else {
+            return false;
+        }
```

Comment:
nit
```suggestion
        }
        return false;
```

Reply:
Feel free to reject if our style says otherwise. 🤷

Reply:
Made fancy with:

```rust
match &self.patches_state.currently_booting_patch {
  Some(currently_booting_patch) => currently_booting_patch.number == patch_number,
  None => false,
}
```